homework3/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace homework3
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Введите значение x:");
+            var x = double.Parse(Console.ReadLine());
+
+            var y = F(x);
+
+            Console.WriteLine("f(x) = " + y);
+            Console.ReadKey();
+        }
+
+        static double F(double x)
+        {
+            return Math.Sqrt(Math.Pow(x, 2) + Math.Sqrt(Math.Pow(x, 2) + 1 / (Math.Pow(x, 2))));
```

Comment:
Квадрат лучше вычислять через произведение. А поскольку вычисляемая величина используется в формуле несколько раз, не стоит каждый раз её вычислять, а лучше запомнить в локальной переменной.
```suggestion
            var y = x * x;
            return Math.Sqrt(y + Math.Sqrt(y + 1 / y));
```